invenio_drafts_resources/services/records/components/base.py

```diff
@@ -128,15 +126,15 @@ def update_draft(self, identity, data=None, record=None, errors=None):
             return  # exit early
 
         if draft_files.enabled and not draft_files.items():
+            my_message = _("Missing uploaded files.")
```

Comment:
I'm still worried about changing this line and the translations. So, the translations files had something for the full sentence (`"Missing uploaded files. To disable files for this record please mark it as metadata-only."`). Now that they are two independent sentences, do we need to split as well the translation?